libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.m

```diff
@@ -494,12 +494,7 @@ - (BOOL)haveValidSession {
 }
 
 - (SFOAuthAdvancedAuthConfiguration) advancedAuthConfiguration {
-    return [SFUserAccountManager sharedInstance].advancedAuthConfiguration;
-}
-
-- (void)setAdvancedAuthConfiguration:(SFOAuthAdvancedAuthConfiguration)advancedAuthConfiguration
-{
-    [SFUserAccountManager sharedInstance].advancedAuthConfiguration = advancedAuthConfiguration;
+    return SFOAuthAdvancedAuthConfigurationAllow;
```

Comment:
I'm assuming this will go away when we remove `SFAuthenticationManager`?

Reply:
@bhariharan  Yes this will go away in the next PR.